sympy/printing/tests/test_latex.py

```diff
@@ -2249,6 +2250,39 @@ def test_KroneckerProduct_printing():
     assert latex(KroneckerProduct(A, B)) == r'A \otimes B'
 
 
+def test_Series_printing():
+    tf1 = TransferFunction(x*y**2 - z, y**3 - t**3, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    tf3 = TransferFunction(t*x**2 - t**w*x + w, t - y, y)
+    assert latex(Series(tf1, tf2)) == \
+        'Series\\left(\\frac{x y^{2} - z}{- t^{3} + y^{3}}, \\frac{x - y}{x + y}\\right)'
+    assert latex(Series(tf1, tf2, tf3)) == \
+        'Series\\left(\\frac{x y^{2} - z}{- t^{3} + y^{3}}, \\frac{x - y}{x + y}, \\frac{t x^{2} - t^{w} x + w}{t - y}\\right)'
+    assert latex(Series(-tf2, tf1)) == \
+        'Series\\left(\\frac{- x + y}{x + y}, \\frac{x y^{2} - z}{- t^{3} + y^{3}}\\right)'
```

Comment:
These should print as mathematical expressions without "Series". Revert to the original tests.